openhub/models.py

```diff
@@ -92,14 +105,20 @@ class OutsideCommitter(models.Model):
     def __str__(self):
         return self.name
 
+    def get_absolute_url(self):
+        """
+        Returns the url to access a particular portfolio project instance.
+        """
+        return reverse('outsidecommitterdetail', args=[str(self.id)])
+
 
 class MostCommit(models.Model):
 
     project = models.CharField(max_length=100)
     commits = models.IntegerField()
 
     def __str__(self):
-        return self.project1
+        return self.project
```

Comment:
separate issue for this.  it is a bug, which needs to be fixed immediately, not as part of this change.  it should be labelled low, and advertise it on gsoc room for someone to fix it for you.